lib/apiservers/engine/backends/archive.go

```diff
@@ -146,7 +146,14 @@ func (c *Container) importToContainer(op trace.Operation, vc *viccontainer.VicCo
 	mounts := mountsFromContainer(vc)
 	mounts = append(mounts, types.MountPoint{Destination: "/"})
 	writerMap := NewArchiveStreamWriterMap(op, mounts, target)
-	defer writerMap.Close() // This should shutdown all the stream connections to the portlayer.
+	defer func() {
+		// This should shutdown all the stream connections to the portlayer.
+		e1 := writerMap.Close()
+		if err == nil {
```

Comment:
Do you need to get the error here?  Working through it in my head, only one io.Copy() occurs at any one time.  If there is an error, the error is returned from io.Copy and then the defer function gets called.  There shouldn't be any other errors from any other writers.